examples/feature_selection/plot_select_from_model_diabetes.py

```diff
@@ -143,3 +143,62 @@
 # attribute. The forward SFS is faster than the backward SFS because it only
 # needs to perform `n_features_to_select = 2` iterations, while the backward
 # SFS needs to perform `n_features - n_features_to_select = 8` iterations.
+
+# %%
+# Selecting Features Using Sequential Feature Selection and Negative tol Values
+# -----------------------------------------------------------------------------
+#
```

Comment:
I think it would be good to add a sentence or two here explaining why you'd use a negative `tol` and briefly describing what will happen. And then dive into the code and demo.